src/mockserver.js

```diff
@@ -1,9 +1,12 @@
 var config = window.__karma__.config,
-	ui5config = (config && config.openui5) || {},
+	ui5config = (config && config.openui5.config) || {},
 	mockserverConfig = ui5config.mockserver || {};
 
 //workaround for crossroads/mockserver issue
 (function() {
+	if (typeof module === undefined) {
+		var module = '';
```

Comment:
See my comment https://github.com/SAP/karma-openui5/pull/2#issuecomment-170025760
I would suggest to fix it like this
```js
var original;
if (typeof module !== "undefined") {
	original = module;
	module = undefined;
}
jQuery.sap.require("sap.ui.core.util.MockServer");
if (typeof original !== "undefined") {
	module = original;
	original = undefined;
}
```